apps/examples/templates/example.jade

```diff
@@ -0,0 +1,16 @@
+if example.channels
+  .Page--examples__example-section.js-example(data-id=example.id)
+    .Page--examples__header
+      .Page--examples__title
+        h4.Type.Type--size-h4 &nbsp;
+      .Page--examples__description
+        h4.Type.Type--size-h4
+          strong #{example.title}:&nbsp;
+          = example.description
+    .Page--examples__channels
+      for channel in example.channels
+        include ../components/channel_group/index
+    if example.channel_ids.length > example.channels.length
+      .Page--examples__button.Utility--align-center
+        a.Button.Button--size-lg.Button--border-heavy.js-load-more( href='#' )
+          | See more #{example.id} channels
```

Comment:
I don't really see how this is a modifier on the `.Page` class. Also variants should always be using the thing it's modifying: `.Page.Page--modifier`